prow/config/jobs/istio.yaml

```diff
@@ -481,6 +481,10 @@ jobs:
     types: [presubmit]
     command: [make, -e, "TARGET_OS=darwin", build]
 
+  - name: windowscnibuildcheck
+    types: [presubmit]
+    command: [make, -e, "TARGET_OS=windows", build-cni]
```

Comment:
Should we try to build more? At least istioctl should build. I guess we find that from release test. The OSX one is largely to ensure developers don't get broken I guess. I suspect we could make windows work more broadly, maybe we start with this though. I would name the job windowsbuildcheck to future proof it from the details

Reply:
Yeah, there are some other refactors that are needed to make windows build more generally. Good call out on the naming though. Will add